frontend/javascripts/viewer/view/context_menu.tsx

```diff
@@ -1726,7 +1828,7 @@ function ContextMenuInner() {
     </FastTooltip>
   );
 
-  const areSegmentStatisticsAvailable = wasSegmentOrMeshClicked && isSegmentIndexAvailable;
+  const areSegmentStatisticsAvailable = false && wasSegmentOrMeshClicked && isSegmentIndexAvailable;
```

Comment:
@fm3 Do we consider fixing this segment stats oom issue? I think we do not have an issue for this yet. At least I could not find any while searching for it